<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIColoreoGrafo</class>
 <widget class="QMainWindow" name="UIColoreoGrafo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>1349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconos.qrc">
    <normaloff>:/iconos/iconoVentana.png</normaloff>:/iconos/iconoVentana.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-image: url(:/iconos/gradient7.png);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButtonVerGrafo">
      <property name="text">
       <string>Ver ejemplo de un grafo</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Presione el siguiente botón para añadir un nuevo vértice:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="buddy">
           <cstring>lineArco</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CargarVertice">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Cargar vértice</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Ingrese aristas separadas por una coma:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="buddy">
           <cstring>lineArco</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineArco">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ejemplo : 1,2 (Agregue previamente los vertices 1 y 2 )&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string extracomment="Ejemplo : 1 "/>
          </property>
          <property name="maxLength">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CargarArco">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Cargar arista</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="borrarVertice">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Borrar último vértice</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="borrarUltimoArco">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Borrar última arista</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BorraTodo">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Borrar todo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="espacio0">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	background-color: transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt;El problema del coloreo de un grafo &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic;&quot;&gt;G&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt;, consiste en &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic;&quot;&gt;asignar un color a cada vertice&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt; de G, de manera que &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic;&quot;&gt;dos vertices conectados por una arista reciban colores distintos&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic;&quot;&gt;		&lt;/span&gt;&lt;img src=&quot;:/iconos/grafo_imagen.png&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic;&quot;&gt;     &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt; Definimos un Grafo &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic;&quot;&gt;G=(V,E) &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt; tal que &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic;&quot;&gt;V&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt; es un &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic;&quot;&gt;conjunto no vacío de vértices&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt; y &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic;&quot;&gt;E &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt;es un c&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic;&quot;&gt;onjunto de aristas&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt;. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt;      Es  un &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic;&quot;&gt;diagrama formado por vértices y aristas &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt;, donde cada uno de estas aristas une dos vértices.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt; font-style:italic;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowserC">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Formalización general&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#000000;&quot;&gt;V : Conjunto de Vértices&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#000000;&quot;&gt;. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#000000;&quot;&gt;E: Conjunto de Aristas.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt; color:#000000;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#000000;&quot;&gt;Para representar mediante literales que un vértice tiene un determinado color (3 colores posibles) usaremos esta notación: &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic; color:#000000;&quot;&gt;{ Color_v : v pertenece a V y  Color pertenece a {Rojo, Azul, Verde} }&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#000000;&quot;&gt;   &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#000000;&quot;&gt;Condiciones : &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#000000;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#000000;&quot;&gt;1) Cada vértice tiene al menos un color. El conjunto de cláusulas con el que representamos esto es :&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#000000;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#0055ff;&quot;&gt;      &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic; color:#0055ff;&quot;&gt; {  {Rojo_v, Azul_v, Verde_v }  : v pertenece a V }&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#0055ff;&quot;&gt;   &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#000000;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#000000;&quot;&gt;2) Cada vértice tiene un solo color y se representa : &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#ff0000;&quot;&gt;      &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-style:italic; color:#ff0000;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic; color:#0055ff;&quot;&gt;{~Rojo_v , ~Azul_v} , {~Rojo_v , ~Verde_v} , {~Azul_v , ~Verde_v} &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#000000;&quot;&gt;   &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#000000;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#000000;&quot;&gt;3) Dos vértices conectados por un mismo arco no tienen el mismo color  : &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#ff0000;&quot;&gt;  &lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#0055ff;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic; color:#0055ff;&quot;&gt; { {~Rojo_a , ~Rojo_b} ,{~Azul_a , ~Azul_b} , {~Verde_a , ~Verde_b } : (a,b) pertenece a E } &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Aristas cargadas:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QTextBrowser" name="textBrowserA">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>602</width>
          <height>93</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lista de Aristas (Si los vertices que se ingresan, no existen, no se creara la arista ).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Vértices cargados:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QTextBrowser" name="textBrowserV">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Lista de vértices&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="espacio2">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	background-color: transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Resolver">
      <property name="toolTip">
       <string/>
      </property>
      <property name="text">
       <string>Ver resolución </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonVerVal">
      <property name="text">
       <string>Ver valuaciones</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="botonGrafos">
      <property name="text">
       <string>Ver grafos solución</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>textBrowser_2</zorder>
   <zorder>textBrowser</zorder>
   <zorder>textBrowserC</zorder>
   <zorder>label_4</zorder>
   <zorder>label_2</zorder>
   <zorder>textBrowserA</zorder>
   <zorder>textBrowserV</zorder>
   <zorder>Resolver</zorder>
   <zorder>pushButtonVerGrafo</zorder>
   <zorder>pushButtonVerVal</zorder>
   <zorder>botonGrafos</zorder>
   <zorder></zorder>
   <zorder>CargarVertice</zorder>
   <zorder></zorder>
   <zorder>espacio0</zorder>
   <zorder></zorder>
   <zorder>espacio2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionProblema_del_Coloreo_de_un_Grafo"/>
    <addaction name="separator"/>
    <addaction name="action_Que_es_un_Grafo"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAtras"/>
   <addaction name="separator"/>
   <addaction name="actionNuevo"/>
   <addaction name="separator"/>
   <addaction name="actionVerFormalizacion"/>
  </widget>
  <action name="action_Que_es_un_Grafo">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/configuration12.png</normaloff>:/iconos/configuration12.png</iconset>
   </property>
   <property name="text">
    <string>¿Que es un Grafo? </string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="icon">
    <iconset>
     <normaloff>add121.png</normaloff>add121.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionAtras">
   <property name="icon">
    <iconset>
     <normaloff>back28.png</normaloff>back28.png</iconset>
   </property>
   <property name="text">
    <string>Atrás</string>
   </property>
  </action>
  <action name="actionProblema_del_Coloreo_de_un_Grafo">
   <property name="icon">
    <iconset>
     <normaloff>configuration12.png</normaloff>configuration12.png</iconset>
   </property>
   <property name="text">
    <string>Problema del Coloreo de un Grafo</string>
   </property>
  </action>
  <action name="actionVerFormalizacion">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/search67.png</normaloff>:/iconos/search67.png</iconset>
   </property>
   <property name="text">
    <string>Ver Formalizacion</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineArco</tabstop>
  <tabstop>textBrowserV</tabstop>
  <tabstop>textBrowserC</tabstop>
 </tabstops>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineArco</sender>
   <signal>returnPressed()</signal>
   <receiver>CargarArco</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
